packages/core/src/components/service-message/ServiceMessageBanner.tsx

```diff
@@ -0,0 +1,103 @@
+import { LayoutAnimation, Linking, View } from "react-native";
+import {
+    EDSStyleSheet,
+    PressableHighlight,
+    Typography,
+    useStyles,
+    CellSwipeItem,
+} from "@equinor/mad-components";
+import { ExpansionToggle } from "./components/ExpansionToggle";
+import { DisableableSwipeable } from "./components/DisableableSwipeable";
+import { useServiceMessageState } from "./ServiceMessageProvider";
+import { resolveMessageFromServiceMessage } from "./utils/resolveMessageFromServiceMessage";
+
+export const ServiceMessageBanner = () => {
+    const styles = useStyles(theme);
+    const {
+        serviceMessage,
+        isDismissed,
+        setIsDismissed,
+        isExpanded,
+        setIsExpanded,
+        expansionEnabled,
+        setExpansionEnabled,
+        isError,
+    } = useServiceMessageState();
+    const dismissAndURLIsVisible = !expansionEnabled || (expansionEnabled && isExpanded);
+    const onPressServiceMessage = () => {
+        LayoutAnimation.configureNext({ ...LayoutAnimation.Presets.easeInEaseOut, duration: 100 });
+        setIsExpanded(isExpanded => !isExpanded);
+    };
+    if (isDismissed) return null;
+    if (!serviceMessage) return null;
+    const { message, url } = resolveMessageFromServiceMessage(serviceMessage, isError);
+    return (
+        <DisableableSwipeable
+            disabled={!dismissAndURLIsVisible}
+            renderRightActions={() => (
+                <CellSwipeItem
+                    title="Remove"
+                    iconName="close"
+                    color="danger"
+                    onPress={() => setIsDismissed(true)}
+                />
+            )}
+        >
+            <PressableHighlight
+                style={styles.container}
+                onPress={onPressServiceMessage}
+                disabled={!expansionEnabled}
+            >
+                <View style={{ flex: 1 }}>
+                    <Typography
+                        color={styles.text.color}
+                        numberOfLines={expansionEnabled && !isExpanded ? 2 : 0}
+                        onTextLayout={e => {
+                            if (e.nativeEvent.lines.length > 2) {
+                                setExpansionEnabled(true);
+                            }
+                        }}
+                    >
+                        {message}
+                    </Typography>
+                    {dismissAndURLIsVisible && (
+                        <>
+                            {url && (
+                                <Typography
+                                    group="interactive"
+                                    variant="link"
+                                    style={styles.text}
+                                    onPress={() => void Linking.openURL(url)}
+                                >
+                                    {url}
+                                </Typography>
+                            )}
+                            <Typography
```

Comment:
The previous text here read "click to remove" or something like that. Now that the text is no longer a link, should we remove the underline / change the styling since it is no longer interactive?

Reply:
@EmilieOlsen What do you think?

Reply:
(Regarding the "Swipe to remove" text)

Reply:
Agree, remove the underline. But keep the boldness. In life.